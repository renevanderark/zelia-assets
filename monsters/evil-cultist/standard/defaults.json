{
    "hp":            50,
    "damage":         4,
    "xAcceleration":  2,
    "yAcceleration":  2
}