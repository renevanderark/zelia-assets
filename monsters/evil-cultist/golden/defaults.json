{
    "hp":            80,
    "damage":         6,
    "xAcceleration":  3,
    "yAcceleration":  3
}